src/applications/personalization/va-profile/components/ProfileView.jsx

```diff
@@ -90,13 +110,15 @@ class ProfileView extends React.Component {
     if (user.profile.verified) {
       if (user.profile.veteranStatus === 'OK') {
         content = (
-          <div>
-            <AlertBox onCloseAlert={message.clear} isVisible={!!message.content} status="success" content={<h3>{message.content}</h3>}/>
-            <Hero fetchHero={fetchHero} hero={hero} militaryInformation={militaryInformation}/>
-            <ContactInformation {...this.props}/>
-            <PersonalInformation fetchPersonalInformation={fetchPersonalInformation} personalInformation={personalInformation}/>
-            <MilitaryInformation fetchMilitaryInformation={fetchMilitaryInformation} militaryInformation={militaryInformation}/>
-          </div>
+          <DowntimeNotification appTitle={appTitle} render={this.handleDowntime} dependencies={[services.emis, services.evss, services.mvi]}>
+            <div>
```

Comment:
Do you need this div?

Reply:
I was seeing this error without that -

```
Error: DowntimeNotification.render(): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.
```

Is there a way around that?